CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 3.9)
 project (OpenMEEG VERSION 2.4.9999 LANGUAGES C CXX)
 
-set(CMAKE_CXX_STANDARD 11)  # use c++11
+set(CMAKE_CXX_STANDARD 17)  # use c++17
```

Comment:
was this really necessary? does an ubuntu or debian stable support this?

Reply:
This is not fully necessary. I can work around its need (it is just that static constexpr unsigned x[]; changed semantics between c++11 and c++17. If we need to stick with c++11, I need to add a new file to define the static member....
I think the support is there for quite some time.... As far as I can tell c++17 support started from g++5 and was mostly complete by g++-7, That does not mean that there are no bugs. I surely do not yet totally understand the ubuntu ci problem.... It fails on a simple c++11 feature.... I need to investigate more.

Reply:
We are currently at gcc-9 and soon gcc-10...